lisa/sut_orchestrator/hyperv/schema.py

```diff
@@ -62,9 +64,27 @@ class VhdSchema(schema.ImageSchema):
     vhd_path: Optional[str] = None
 
 
+@dataclass_json()
+@dataclass
+class DevicePassthroughSchema:
```

Comment:
Similar class is implemented for Libvirt platform. This can be made common and moved into utils.

Reply:
member type is different.

Reply:
I see only additional member (`managed: str = ""`) in Libvirt;s `DevicePassthroughSchema`. Otherwise its same for `hyperv` platform.
You can either ignore the usage of additional member in hyperv platform or we can create something called `BaseDevicePassthroughSlchema` class in common utils and if needed Libvirt can inherit it.

Reply:
Done